gamemodes/cinema/gamemode/modules/scoreboard/cl_settings.lua

```diff
@@ -174,6 +174,8 @@ function SETTINGS:Create()
     --addCheckbox('Don\'t load chat images',"fedorachat_hideimg","Hides all images in chat")
     addCheckbox('Hide interface', "cinema_hideinterface", "Clean Your Screen")
     addCheckbox('Hide players in theater', "cinema_hideplayers", 'For when trolls stand in front of your screen')
+    addCheckbox('Trust Videos', "cinema_trust_videos", 'Don\'t show the warning on untrusted videos')
```

Comment:
This setting is probably just going to be confusing. How about instead, it shows "run cinema_load_all_urls 1 to never see this again" on the warning overlay? Its in cl_draw.lua